action.yml

```
@@ -0,0 +1,76 @@
+name: 'Run quickstart'
+description: 'Runs quickstart docker image'
+inputs:
+  version:
+    description: "Version (docker tag) of quickstart image to use"
+    required: true
+    default: "latest"
```

Suggest rename so field is 1-1 with the use.
```suggestion
  tag:
    description: "Image tag of quickstart image to use"
    required: true
    default: "latest"
  image:
    description: "Image for the quickstart image to use"
    required: true
    default: "docker.io/stellar/quickstart"
```